### PR TITLE
Bugfix/concurrentlog

### PR DESCRIPTION
**Pull request by codytipton**
Provide class ConcurrentRotatingFileHandler and import concurrent_log_handler before logging.FileConfig()

2nd of 3 PR's addressing #63 
Do not merge before https://github.com/Netuitive/omnibus-netuitive-agent/pull/104

### CODE REVIEW

**Review by TheConnMan — CHANGES_REQUESTED** [review]
LGTM, just the whitespace nit pick.

File: src/diamond/logging/handlers/concurrentloghandler.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+class ConcurrentRotatingFileHandler(TRFH):
+
+    def flush(self):
+        try:
+            super(ConcurrentRotatingFileHandler, self).flush()
+        except IOError:
+            sys.stderr.write('ConcurrentRotatingFileHandler received a IOError!')
+            sys.stderr.flush()
+
```
Comment (rclagett) [inline src/diamond/logging/handlers/concurrentloghandler.py]: Can you remove the whitespace here?

**Review by rclagett — CHANGES_REQUESTED** [review]

File: src/diamond/logging/handlers/concurrentloghandler.py (hunk 1 above)
Comment (rclagett) [reply src/diamond/logging/handlers/concurrentloghandler.py]: Sorry, I didn’t see @TheConnMan mentioned that already.

Comment (codytipton) [reply src/diamond/logging/handlers/concurrentloghandler.py]: Done 👍

**Review by rclagett — APPROVED** [review]

**Review by Grumblesmith — APPROVED** [review]

**Pull request merged**